src/indexer/merger.rs

```diff
@@ -17,6 +17,12 @@ use store::StoreWriter;
 use std::cmp::{max, min};
 use termdict::TermDictionary;
 use termdict::TermStreamer;
+use schema::FieldType;
+use termdict::TermOrdinal;
+use schema::Cardinality;
+use std::collections::HashMap;
+use fastfield::MultiValueIntFastFieldReader;
+use std::cmp;
```

Comment:
nit: Already importing `cmp::{max, min}`. Side-note: `val.min(other_val)` also exists now.

Also imports in general need some re-ordering. Do we use `cargo fmt`?

Reply:
changed btw. I'll run cargo fmt once I integrate with master. Running it now might make the merge painful